javascript/fluent.js

```diff
@@ -125,16 +125,29 @@
 				} else {
 					label.attr('title', 'Using default locale value');
 				}
+			},
+			/**
+			 * The JSON in the data attribute will be encoded, so this is a bit of a hack to decode it before parsing
```

Comment:
If you view the source you'll see something like this: `data-default-locale-value="&quot;i&#039;m a default value&quot;" />`. This is a result of `json_encode('I'm a default value')`, which would be `"I'm a default value"`, yet when it's inserted into the DOM it gets encoded as such (assuming by the SS framework)